drivers/SmartThings/philips-hue/src/handlers/commands.lua

```diff
@@ -92,6 +92,7 @@ local function do_switch_level_action(driver, device, args)
     return
   end
 
+  --Note: this will always be 0 due to state cache changes
```

Comment:
comment is not quite right since it won't be 0.  I assume it will be `nil`

Reply:
This change will lead to us sending an `on` to the light with every switch level capability command. I'd like to think about storing this piece of state in a field as well to avoid that extra round trip. It could add up for automations with lots of lights.

Reply:
I changed this to be a persisted field, set when we emit on/off events.